pom.xml

```diff
@@ -22,9 +22,14 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <version>4.13.1</version>
```

Comment:
Why you need to change this?

Reply:
IDE prompted to avoid warnings) In this case, can the upgrade version spoil the result?

Reply:
@SalnikovMarkiian  no, but you shouldn't add irrelevend changes to your PRs - this is considered a bad tone :)